src/features/collections/MatchPane.tsx

```diff
@@ -0,0 +1,226 @@
+import {
+  createMatch,
+  getCollectionMatchers,
+  getMatch,
+} from '../../common/api/collectionsApi';
+import styles from './Collections.module.scss';
+import { useCallback, useEffect, useMemo, useState } from 'react';
+import { snakeCaseToHumanReadable } from '../../common/utils/stringUtils';
+import { Select, SelectOption } from '../../common/components';
+import { listObjects, listWorkspaceInfo } from '../../common/api/workspaceApi';
+import { parseError } from '../../common/api/utils/parseError';
+import { useAppParam, useUpdateAppParams } from '../params/hooks';
+
+export const CollectionMatchPane = ({
+  collectionId,
+}: {
+  collectionId: string;
+}) => {
+  const matchId = useAppParam('match');
+
+  return (
+    <>
+      <h3>Match Options</h3>
+      {matchId ? <ViewMatch /> : <CreateMatch collectionId={collectionId} />}
+    </>
+  );
+};
+
+const ViewMatch = () => {
+  const matchId = useAppParam('match');
+  const updateAppParams = useUpdateAppParams();
+
+  const matchQuery = usePollMatch(matchId);
+  const match = matchQuery.data;
+
+  const matchCount =
+    match?.match_state === 'complete' ? match.matches.length : 0;
+  const upaCount =
+    match?.match_state === 'complete'
+      ? match.upas.flatMap((upaList) => upaList.split(';')).length
+      : 0;
+
+  const handleClear = () => {
+    updateAppParams({ match: null });
+  };
+
+  return (
+    <div>
+      {matchQuery.isLoading ? (
+        'Loading...'
+      ) : (
+        <ul>
+          <li>Match ID: {match?.match_id}</li>
+          <li>Match Status: {match?.match_state}</li>
+          {match?.match_state === 'complete' ? (
+            <li>
+              You input a total of <strong>{upaCount}</strong> data objects,
+              matching <strong>{matchCount}</strong> collection items.
+            </li>
+          ) : null}
+        </ul>
+      )}
+      <button onClick={handleClear}>Clear Match</button>
+    </div>
+  );
+};
+
+const CreateMatch = ({ collectionId }: { collectionId: string }) => {
+  const updateAppParams = useUpdateAppParams();
+  const matchersQuery = getCollectionMatchers.useQuery(collectionId);
+  const matchers = matchersQuery.data?.data;
+  const [matcherSel, setMatcherSel] = useState<SelectOption | undefined>();
+  const matcherSelected = matchers?.find((d) => d.id === matcherSel?.value);
+  const matcherOptions: SelectOption[] = useMemo(() => {
+    setMatcherSel(undefined);
+    return (
+      matchers?.map((matcher) => ({
+        value: matcher.id,
+        label: `${snakeCaseToHumanReadable(matcher.id)}: ${
```

Comment:
re: @MrCreosote 

Don't use the label from the server, or add user-oriented labels server-side and use those here

Reply:
As label name/desc aren't user-facing atm